server/controllers/listing.js

```diff
@@ -19,13 +19,26 @@ class ListingController {
 
   getAllListings = async (req, res, next) => {
     try {
-      const listing = await this.listingService.getAllListings(req.params);
-   
-      res.status(200).json(listing);
+      const listings = await this.listingService.getAllListings(req.params);
+      res.status(200).json(listings);
     } catch (err) {
       next(ApiError.internal(`${err}`));
     }
   };
+
+  getOneListing = async (req, res, next) => {
+    try{
+      const listingId = req.params.id;
+      const listing = await this.listingService.getOneListing(req.params.id);
+      if(listing == null){
+        next(ApiError.notFound(`Listing with id ${listingId} is not found.`));
+        return; 
```

Comment:
I believe there was a style change for these types of functions in #75 where the empty return statements were removed

Reply:
Yep! @AzizulHakim96 just remove the retuen on line 35!